docs/HowToFAQ/FAQSplitHere.md

```diff
@@ -0,0 +1,34 @@
+---
+title: How do `{:split_here}` and `{:focus}` work to divide up a verification condition?
+---
+
+## Question
+
+How do `{:split_here}` and `{:focus}` work to divide up a verification condition?
```

Comment:
Well, the descriptions are in the section on attributes, which describes each attribute one by one.
And there are lots of questions on Slack which are essentially "how does X work?" -- some before and some after an update of the RM.

Reply:
Okay, but still, why would the answer to "How does feature X work?" be something other than "Read the reference manual on feature X [here]"  ?

Reply:
Well, because a curt 'RTFM' is a bit rude and not entirely helpful :-) I did put in links to the relevant sections of the RM, but also decided to put in a short summary of the difference.

Reply:
Okay, but the RM should be the best place to read about these features right? I'm a little worried that if we're duplicating information we'll have a race between the two locations, each improving over time but none be strictly better than the other.

Reply:
Yes the RM should be the "final word"